src/RFC-0122_Burning.md

```diff
@@ -0,0 +1,193 @@
+This document and its content are intended for information purposes only and may be subject to change or update
+without notice.
+
+This document may include preliminary concepts that may or may not be in the process of being developed by the Tari
+community. The release of this document is intended solely for review and discussion by the community of the
+technological merits of the potential system outlined herein.
+
+## Goals
+
+The aim of this Request for Comment (RFC) is to describe the process of Burning UTXOs and to track the amount burned. 
+
+## Related Requests for Comment
+
+* [RFC-0100: Base Layer](RFC-0100_BaseLayer.md)
+* [RFC-0120: Consensus](RFC-0120_Consensus.md)
+
+## Description
+
+Blockchains have used the burn method to destroy coins in circulation and block their use forever. Most chains use a reclaimable address
+to denote burned coins. Using [RFC-0201: TariScript](RFC-0201_TariScript.md), Tari can also use this method, but this does not explicitly
+remove the coins from circulation. This RFC details a method to remove coins from circulation permanently.
+
+
+## Introduction
+
+To completely remove coins from circulation, we need to mark outputs as burned, and we need to change the balance equation to
+allow pruned nodes and non pruned nodes to still verify the integrity and emission of Tari.
+
+### Transaction output changes
+
+Currently, each transaction output has a field called [OutputFeatures] that tracks the unique properties of each output. 
+Inside this field, we track every possible type of output. We need to add another type here called burned. 
+
+```rust,ignore
+pub enum OutputType {
+    /// An standard non-coinbase output.
+    Standard = 0,
+    /// Output is a coinbase output, must not be spent until maturity.
+    Coinbase = 1,
+    /// This output is burned and may never be used as a transaction input.
+    Burned = 2,
+}
+```
+
+### Blockheader changes
+
+On the header of each block, we need to add a field for tracking the sum of all burned outputs on that block.
+
+```rust,ignore
+pub struct BlockHeader {
+    /// Version of the block
+    pub version: u16,
+    ...
+    /// Sum of kernel offsets for all kernels in this block.
+    pub total_kernel_offset: BlindingFactor,
+    /// Sum of script offsets for all kernels in this block.
+    pub total_script_offset: BlindingFactor,
+    /// Sum of all burned output commitments in this block.
+    pub burned_total: Commitment,
+    /// Nonce increment used to mine this block.
+    pub nonce: u64,
+    /// Proof of work summary
+    pub pow: ProofOfWork,
+}
+```
+
+This field must contain the sum of all the commitments of all outputs in the block marked as spent. 
+
+$$
+\begin{align}
+burned\_total = \sum_j\mathrm{C_{j}}  \\\\
+\text{for burned output}, i, \\\\
+\end{align}
+\tag{1}
+$$
+
+### Block propagation consensus rules changes
+
+When a block is received, a base node will typically check the following equation to ensure that the emission is correct:
```

Comment:
Thi Latex doesn't seem to render in Github's markdown, but it might be fine in mdbook

Reply:
Yeah github does not render correctly